mkdocs.yml

```diff
@@ -70,7 +70,7 @@ plugins:
 extra:
   links:
     portal: !!python/object/apply:os.getenv ['DEV_PORTAL_SITE']
-    api: !!python/object/apply:os.getenv ['RESTDAPI_DOCS']
```

Comment:
Its worth it to make sure this wasn't used across the code

Reply:
I got no results for a search for `link.api`...